src/style.css

```diff
@@ -0,0 +1,121 @@
+@font-face {
+  font-family: Avenir;
+  src: url("./fonts/Avenir-Book.ttf");
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url("./fonts/Avenir-Heavy.ttf");
+  font-style: normal;
+  font-weight: 800;
+}
+
+body {
+  font-family: Avenir, sans-serif;
+}
+
+.container-big-bar {
+  position: relative;
+  display: block;
+  margin-top: 20px;
+  margin-bottom: 20px;
+  height: 68px;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  background: #fff;
+}
+
+.big-search-bar {
+  position: absolute;
+  box-sizing: border-box;
+  left: 62px;
+  border: none;
+  top: 23px;
+  height: 22px;
+  margin: auto;
+  font-family: inherit;
+  font-size: inherit;
+  font-weight: bold;
+  color: #3d4e61;
+  text-shadow: 0 4px 4px rgba(0, 0, 0, 0.25);
+  line-height: 22px;
+}
+
+.big-search-bar:focus {
+  outline: none;
+}
+
+.search-logo {
+  width: 19;
+  height: 19;
+  position: relative;
+  top: 24px;
+  left: 25px;
+}
+
+.big-search-bar::placeholder {
+  font-family: inherit;
+  font-weight: 300;
+  text-shadow: none;
+  line-height: 22px;
+  color: #3d4e61;
+  font-size: 16px;
+}
+
+.container-big-bar:hover {
+  box-shadow: 0 3px 8px;
+}
+
+.container-small-bar {
+  position: relative;
+  height: 40px;
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+}
+
+.search-small-logo {
+  position: absolute;
+  width: 11px;
+  height: 11px;
+  top: 14px;
+  left: 12px;
+}
+
+.small-search-bar {
+  position: absolute;
+  font-family: inherit;
+  box-sizing: border-box;
+  border: none;
+  margin: auto;
+  height: 20px;
+  top: 10px;
+  left: 33px;
+  font-weight: 900;
+  font-size: 14px;
+  line-height: 19px;
+  color: #3d4e61;
+}
+
+.small-search-bar::placeholder {
+  font-size: 14px;
+  font-family: inherit;
+  /* text-shadow: 0 4px 4px rgba(0, 0, 0, 0.25); */
+  font-weight: 400;
+  color: #3d4e61;
+  line-height: 19px;
+}
```

Comment:
Repeatable styles with `.big-search-bar::placeholder`. You can optimize your code